src/pages/simulator/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { Link } from '@tanstack/react-router';
+import { Button } from 'components/common/button';
+import { Page } from 'components/common/page';
+import { useModal } from 'hooks/useModal';
+import { SimulatorSearch } from 'libs/routing';
+import { Token } from 'libs/tokens';
+import { useState } from 'react';
+
+const defaultParams: SimulatorSearch = {
+  start: '1672491600',
```

Comment:
I assume you don't plan to keep those - but of course the dates can be derived from Date::now